operator/controllers/btpoperator_controller_test.go

```diff
@@ -4,16 +4,20 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	appsv1 "k8s.io/api/apps/v1"
 	"os"
 	"sync"
 	"time"
 
+	appsv1 "k8s.io/api/apps/v1"
+
 	"github.com/kyma-project/btp-manager/operator/api/v1alpha1"
-	"github.com/kyma-project/module-manager/operator/pkg/types"
+	"github.com/kyma-project/btp-manager/operator/internal/gvksutils"
+	ymlutils "github.com/kyma-project/btp-manager/operator/internal/ymlutils"
+	"github.com/kyma-project/module-manager/pkg/types"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	. "github.com/onsi/gomega/gstruct"
+	cp "github.com/otiai10/copy"
```

Comment:
why is this package required? We can achieve the same result by using standard library packages